package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "sofie-tv2-server",
+  "version": "0.0.0",
+  "repository": "git@github.com:tv2/sofie-tv2-server.git",
+  "author": "Anders Frederik Jørgensen <afjo@tv2.dk>",
```

Comment:
Do we want authors?

Reply:
I don't think we want a single author. I have moved it to contributors.